pages/blog/creating-consistency-announcing-asyncapi-spectral-together.md

```diff
@@ -0,0 +1,206 @@
+second anniversary.
+
+## Why AsyncAPI?
+
+While OpenAPI remains one of the top specifications, AsyncAPI is growing. And each has their own [unique benefits and
+challenges](https://www.asyncapi.com/blog/openapi-vs-asyncapi-burning-questions). The ability to have options gives Spectral users more flexibility for their APIs.
+
+Spectral had primarily been an OpenAPI linter that was simply capable of working with other JSON documents. Now, by
+enabling AsyncAPI rulesets, organizations can achieve the consistency they’re looking for with more flexibility.
+
+## Introducing AsyncAPI & Spectral - Together!
+
+The release of [AsyncAPI ruleset](https://meta.stoplight.io/docs/spectral/ZG9jOjUzNDg-async-api-rules) was more than just another feature added; it was considered a milestone. Previously,
+Spectral lacked meaningful rulesets for other formats, and the number of learning resources was somewhat limited, making
+the adaption of Spectral outside of OpenAPI rather minimal.
+
+Our collaboration with AsyncAPI means that we’ve given the appropriate AsyncAPI maintainers write access to the Spectral
+repo AsyncAPI rulesets, though here at Stoplight we will still continue to operate with PRs, and ensure these get
+priority.
+
+### The Game Changer
+
+The introduction of support for another major API definition helped us further assess its level of importance as well as
+initiated a notable shift in the way we perceived Spectral internally. Treating AsyncAPI ruleset as a first-class
+citizen pushed us to entirely decouple our codebase from OpenAPI. The core purpose of Spectral (linting API definitions)
+remained unchanged and still holds the truth, but from an engineering standpoint, it meant a shift in focus to the other
+specs and a renewed attention on writing code that can be applicable everywhere.
+
+Back in 2018 or 2019, plenty of code was oriented around OpenAPI, tests, and naming. All were focused solely on OpenAPI,
+meaning rulesets themselves couldn’t be applied easily to other formats (Spectral didn’t support them previously).
+
+With the release of [Spectral 6](https://blog.stoplight.io/api-linting-with-spectral-keeps-getting-better), that transition period wrapped up and Spectral is now more flexible than it’s ever been.
+
+## Steps to Utilizing AsyncAPI & Spectral
+
+Linting AsyncAPI documents is quite similar to linting OpenAPI documents, meaning
+most of the information about Spectral you’ll find on the internet will apply to AsyncAPI. Together with AsyncAPI
+initiative, we [maintain a Spectral ruleset](https://meta.stoplight.io/docs/spectral/ZG9jOjUzNDg-async-api-rules) containing a number of useful rules you can leverage in your workflow to help
+you get started.
+
+**Step One:**
+
+To get started, you need to install Spectral first, which can be done [here](https://meta.stoplight.io/docs/spectral/docs/getting-started/2-installation.md). Note that you need to have [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) or
```

Comment:
```suggestion
To get started, you need to [install Spectral first](https://meta.stoplight.io/docs/spectral/docs/getting-started/2-installation.md). Note that you need to have [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) or
```

Same **Accessibility** feedback as added in **Step 3**:
https://github.com/asyncapi/website/pull/663/files#r849819370